cli/src/main.rs

```diff
@@ -114,8 +137,26 @@ fn parse_args() -> StrResult<Command> {
 
         let root = args.opt_value_from_str("--root").map_err(|_| "missing root path")?;
         let watch = args.contains(["-w", "--watch"]);
-        let (input, output) = parse_input_output(&mut args, "pdf")?;
-        Command::Compile(CompileCommand { input, output, watch, root })
+        let output_type = if args.contains("--image") {
+            OutputType::Png {
+                pixel_per_pt: args
+                    .opt_value_from_str("--pixel-per-pt")
+                    .map_err(|_| "missing pixel per pt")?
```

Comment:
ma_err can be removed if you're using unwrap_or right after

Reply:
thanks! but after the clap changes this isn't used anymore